net-analyzer/hexinject/hexinject-1.6.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Network packet sniffer and injector"
+HOMEPAGE="http://hexinject.sourceforge.net/"
+SRC_URI="http://downloads.sourceforge.net/project/${PN}/${P}/${P}.tar.gz"
```

Comment:
SourceForge has a special mirror handling support in Gentoo. Please use *mirror://sourceforge* notation.